src/modules/Base/index.tsx

```diff
@@ -0,0 +1,14 @@
+import { useDispatch } from 'react-redux';
+import { Outlet } from 'react-router-dom';
+import { getUserProfileAdaptor } from 'src/core/adaptors';
+import { setUserProfile } from 'src/store/reducers/user.reducer';
+
+const Base = () => {
+  const dispatch = useDispatch();
```

Comment:
check this later and persist `userProfile` instead creating a new layer (if it is possible)